.github/workflows/test-js-packages.yml

```diff
@@ -519,12 +519,25 @@ jobs:
       fail-fast: false
       matrix:
         project:
-        # Disabled as these are currently failing with many visibility errors
-        - { repo: AztecProtocol/aztec-nr, path: ./ }
```

Comment:
What exactly happened with this?

Reply:
Ah it moved further down into a `path`.

Reply:
Yeah, I noticed that aztec sometimes pushes out broken commits to this repository so we need to pull from aztec-packages.